index.html

```diff
@@ -35,9 +35,290 @@ <h3>I am a software developer</h3>
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
+                                    </section>
+                                </section>
+
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <section class="tags">
+                                        <ul class="tags1">
+                                            <li>HTML</li>
+                                        </ul>
+                                        <ul class="tags2">
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
+                                    </section>
+                                </section>
+
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
+                <p class="primary">About Me</p>
+                <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=""></section>
+                <section class="button">
+                    <p class="get">Get My Resume</p>
```

Comment:
- [ ] Please use a `button` or `a` tag for all your buttons. This is best HTML practice.